src/application-insights/azext_applicationinsights/commands.py

```diff
@@ -60,6 +70,10 @@ def load_command_table(self, _):
         g.custom_command('delete', 'delete_component')
         g.custom_command('update-tags', 'update_component_tags')
 
+    with self.command_group('monitor app-insights component billing', command_type=components_billing_sdk, custom_command_type=components_billing_custom_sdk) as g:
+        g.custom_command('update', 'update_componet_billing')
```

Comment:
Do we need generic update to support 'patch' as it seems SDK uses 'put'?

Reply:
good catch. I will make it as patch-like. However, I don't think we need to use generic update. The model contains about six fields. But I can only update two of them. I think exposing the other arguments to the customers may cause issues.

Reply:
Make sense to me.

---
In reply to: [383775771](https://github.com/Azure/azure-cli-extensions/pull/1316#discussion_r383775771) [](ancestors = 383775771)